pages/pfpcollections.tsx

```diff
@@ -1,42 +1,93 @@
-import React from "react";
+import React, { useState } from "react";
 import type { NextPage } from "next";
 import styles from "../styles/pfpcollections.module.css";
 import PfpNftCard from "../components/pfpcollections/pfpNftCard";
 import { ourNfts, NftCollections } from "../utils/constants";
 
 const PFPCollections: NextPage = () => {
+  const [tab, setTab] = useState(0);
+
   return (
     <div className={styles.container}>
-      <div className={styles.gallery}>
-        <section>
-          <p className={styles.subtitle}>Starknet ID Ecosystem</p>
-          <h2 className={styles.title}>PFP collections</h2>
-          <div className={styles.nfts}>
-            {ourNfts.map((collection, index) => (
-              <PfpNftCard
-                key={index}
-                image={collection.imageUri}
-                name={collection.name}
-                onClick={() => window.open(collection.infoPage)}
+      <div className="grid w-full grid-cols-1 md:grid-cols-[minmax(100px,1fr)_5fr] gap-4">
+        <div className={styles.gallery}>
+          <div className="flex flex-col items-center md:items-start gap-[24px] lg:gap-4">
+            <div className="flex gap-2 lg:hidden">
+              <img src="/icons/avatar.svg" />
```

Comment:
Reuse this component: components/domains/steps/registerSteps.tsx (you'll need to make it more modular)